### PR TITLE
Clear the console in browser on each HMR

### PR DESCRIPTION
**Pull request by ameerthehacker**
## What was missing ?

Whenever there was an HRM the console in the browser had old logging data #1064 

## What I have added ?

I have modified the **hmr-runtime.js** file to include a **console.clear()** statement to clear the console whenever there is an HRM

## Little workaround I used

If I used console.clear() some of the test were breaking complaining that **Error: TypeError console.clear function is undefined** I think that the testing environment does not support **console.clear()** function so I replaced the statement in such a way that it will be called only if it is supported.

```JavaScript
console.clear && console.clear()
```

### CODE REVIEW

File: src/builtins/hmr-runtime.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ if ((!parent || !parent.isParcelRequire) && typeof WebSocket !== 'undefined') {
           hmrAccept(global.require, asset.id);
         }
       });
+      // Clear the console after HMR
+      // Check if console.clear function is supported, absence of this caused tests to fail
+      console.clear && console.clear();
```
Comment (DeMoorJasper) [inline src/builtins/hmr-runtime.js]: Probably better to use a mock function for the tests like i explained in #1074

Comment (ameerthehacker) [reply src/builtins/hmr-runtime.js]: Sure @DeMoorJasper I will use a mock function to fix it.

Comment (ameerthehacker) [reply src/builtins/hmr-runtime.js]: Hi @DeMoorJasper I have mocked the function console.clear() in test

**Comment by ameerthehacker** [comment]
@DeMoorJasper I have done the changes you requested please do review

**Comment by ameerthehacker** [comment]
@DeMoorJasper Is there anything else I need to do to get my pull request merged

**Comment by DeMoorJasper** [comment]
@ameerthehacker No it should be fine, but as in the discussion it might not get merged as some developers might not want parcel clearing their console.

**Comment by ameerthehacker** [comment]
@DeMoorJasper that is great

**Pull request merged**